cmd/vela-influx/main.go

```diff
@@ -30,97 +31,123 @@ func main() {
 	fmt.Fprintf(os.Stdout, "%s\n", string(bytes))
 
 	// create new CLI application
-	app := cli.NewApp()
-
 	// Plugin Information
-
-	app.Name = "vela-influx"
-	app.HelpName = "vela-influx"
-	app.Usage = "Vela Influx plugin for for sending data to an InfluxDB"
-	app.Copyright = "Copyright 2021 Target Brands, Inc. All rights reserved."
-	app.Authors = []*cli.Author{
-		{
-			Name:  "Vela Admins",
-			Email: "vela@target.com",
+	cmd := cli.Command{
+		Name:      "vela-influx",
+		Usage:     "Vela Influx plugin for for sending data to an InfluxDB",
+		Copyright: "Copyright 2021 Target Brands, Inc. All rights reserved.",
+		Authors: []any{
+			&mail.Address{
+				Name:    "Vela Admins",
+				Address: "vela@target.com",
+			},
```

Comment:
interesting, we've been kind of just dropping the meta information in other refactors: see https://github.com/go-vela/worker/commit/96d569b1dade2a7c43c3719338debc82d58e1cb4#diff-9a3d5c578376528744f357371d5d528f324b3feaee3942a0aa42315c90292ea7 for example. i don't mind this though, but makes me want to go to these other places and keep it consistent 🥲